mergify_engine/tests/unit/rules/test_parser.py

```diff
@@ -128,6 +132,27 @@
             },
         ),
         ("locked", {"=": ("locked", True)}),
+        (
+            "updated-at>=18:02",
+            {">=": ("updated-at", now + datetime.timedelta(hours=18, minutes=2))},
+        ),
+        (
+            f"updated-at<={now.isoformat()}",
+            {"<=": ("updated-at", now)},
+        ),
+        (
+            "updated-at<=7 days",
+            {"<=": ("updated-at", now + datetime.timedelta(days=7))},
```

Comment:
```suggestion
            "updated-at<=7 days",
            {"<=": ("updated-at", now + datetime.timedelta(days=7))},
```

I don't think this is right. It means it gets translated to `update-at <= now + 7 days`.

If you write `updated-at<=7 days` you mean: `updated-at` is between now and 7 days ago which means:
`now-7days<=update-at<now()`. The `<now()` is obviously true, so we can skip it so it boils down to `now -  7 days <= updated-at` which is equivalent to `now <= updated_at + 7 days`.

Reply:
:facepalm: 
```patch
- +
+ - 
```

Reply:
Actually, it's not just changing the sign. We're not talking about the same things.

1. When I write `updated-at <= 7 days` I mean "updated-at is less than 7 days old" which means: `now - 7 days <= updated-at`

2. When I wrote `updated-at <= 7 days ago` I mean "updated-at date is before the day it was 7 days ago" which translates to `updated-at <= now - 7 days`.

The problem here is that the language says 1 but the code runs 2.

We might need 2 different syntaxes here to make things very clear. It's not clear to me.